src/styles/system/applications/sidebar/tabs/chat.css

```diff
@@ -10,6 +11,17 @@
     footer {
       button {
         margin-top: 3px;
+        &.apply-effect {
+          /* Status conditions should be inverted to dark on the light mode chat */
+          .themed.theme-light &[data-type="status"] {
+            img {
+              filter: invert(1);
+            }
```

Comment:
This is where svgs and some css variables (that depend on light/dark mode) would be very handy. For a follow-up PR.